src/functions.php

```diff
@@ -380,17 +380,17 @@ function copy_to_stream(
         return;
     }
 
-    $bytes = 0;
-    while (!$source->eof()) {
-        $buf = $source->read($maxLen - $bytes);
-        if (!($len = strlen($buf))) {
+    $bufferSize = 8192;
```

Comment:
This buffer size is much smaller than the 1048576 used when reading all of the stream. I don't see why?